sources/web/datalab/polymer/components/datalab-sidebar/datalab-sidebar.html

```diff
@@ -68,7 +68,7 @@
       .iron-selected > .sidebar-button > * {
         color: var(--selection-fg-color);
       }
-      @media only screen and (max-width: 970px) {
+      @media only screen and (max-width: 800px) {
```

Comment:
I saw 800 for the value that is now _addToolbarCollapseThreshold, which I assumed meant 800px. But if there is not an easy way to share a constant, then even if it were the same value, it would not be worth it for this one line. So OK as is.